### DISCUSSION TITLE
How to create users with command line in orchard core?

### DISCUSSION THREAD

**Discussion by uussoft** [OrchardCMS/OrchardCore#6977] [Q&A]
How to create users with command line in orchard core? thanks!

**Comment by BTityo** [comment]
I'm interested too...
How can I run a command from CLI, for example "createUser" command:
https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore.Modules/OrchardCore.Users/Commands/UserCommands.cs#L11

Thanks!

**Comment by ns8482e** [comment]
You can use command in recipes 

**EDIT**
Commands in OrchardCore are not meant to run via CLI on dev machine instead they can be executed via Recipe on hosted server.
You can use `createUser`  command to create new user using recipe. For example

```json
{
    "name": "CreateUser",
    "displayName": "Create User",
    "description": "Creates User",
    "author": "Niraj Soni",
    "website": "https://www.surevelox.com",
    "version": "1.0",
    "issetuprecipe": false,
    "categories": [ "CommandLine", "CLI" ],
    "tags": [ "cli" ],
    
    "steps": [
        { 
            "name" : "Command",
            "Commands" : [
                "createUser /UserName:testuser /Password:PA$$w0rd /Email:testuser@mytestorg.com /Roles:Editor"
            ]
        }
    ]
  }
```

**Reply by BTityo** [reply]
Thank you, how can I use?
Can you share a sample code?

**Reply by ns8482e** [reply]
See above updated answer with sample recipe,

**Reply by BTityo** [reply]
I understand...so we can only use commands in recipes.
Thank you very much!
